sigs.yaml

```diff
@@ -1518,6 +1518,13 @@ sigs:
       name: Tim Hockin
       company: Google
   meetings:
+  - description: Network Policy API Meeting
+    day: Monday
+    time: "13:00"
+    tz: PT (Pacific Time)
+    frequency: weekly
+    url: TBD
```

Comment:
I think one of the SIG leads has to setup a reoccuring zoom call and send me a link?

Reply:
@thockin can you do the honors here since your assigned :)

Reply:
Curiously, I don't have the Zoom creds.  @dcbw or @caseydavenport ?

Reply:
@thockin that is curious! Since you're the one who gave them to me :D 

I can set this up.

Reply:
Should be a calendar invite and zoom meeting available now: https://zoom.us/j/96264742248

Reply:
Can confirm the calendar invite appears here, thank you Casey. I'll cancel the non official one